src/jit/gentree.h

```diff
@@ -2826,6 +2831,8 @@ struct GenTreeCall final : public GenTree
     { 
 #ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
         return varTypeIsStruct(gtType) && !HasRetBufArg(); 
+#elif defined(_TARGET_X86_) && !defined(LEGACY_BACKEND)
```

Comment:
There are multiple places where HasMultiRegRetVal is called that, I think, could potentially include LEGACY_BACKEND. Are we sure we want LEGACY_BACKEND to potentially return true in this function?